cmake/compilerFlags.cmake

```diff
@@ -1,5 +1,9 @@
 # These flags applies to exiv2lib, the applications, and to the xmp code
 
+if ( EXIV2_BUILD_USE_C++11 ) 

```

Comment:
I think we wound need to following. Replace these lines by:

```
if ( EXIV2_BUILD_USE_C++11 ) 
  set(CMAKE_CXX_STANDARD 11)
else()
  set(CMAKE_CXX_STANDARD 98)
endif()

set(CMAKE_CXX_STANDARD_REQUIRED ON)	
set(CMAKE_CXX_EXTENSIONS ON)
```

And more important is to remove all the usages of:

```
target_compile_features(XXX PRIVATE cxx_std_98)
```